aws/ami/windows/windows.pkr.hcl

```diff
@@ -57,7 +57,7 @@ build {
 
   # Install the Visual Studio 2019
   provisioner "powershell" {
-    environment_vars = ["INSTALL_WINDOWS_SDK=1", "VS_YEAR=2019", "VS_VERSION=16.11.21"]
+    environment_vars = ["INSTALL_WINDOWS_SDK=1", "VS_YEAR=2019", "VS_VERSION=16.11.21", "VS_UNINSTALL_PREVIOUS=1"]
```

Comment:
Although the introduced `VS_UNINSTALL_PREVIOUS` variable logic breaks the current behavior, IMO, it's better to keep the VS installation by default. It does not seem that the `Install-VS.ps1` script is used anywhere else so it should not be an issue anyway.

Reply:
@Blackhex should we add VS_UNINSTALL_PREVIOUS=0 to Visual Studio 2022 install ?